utils/gha-runners/createTestconfig.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2014-2022, Intel Corporation
+
+#
+# createTestconfig.sh - Script for creating testconfig files for test excution.
```

Comment:
we are not using camel case for sh file names, please look at other files in the repository